core/templates/components/angular-html-bind/angular-html-bind-wrapper.directive.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
Should the file name be .component.ts?

Reply:
No, it is a directive :). Line 47 of this file: https://github.com/oppia/oppia/pull/12138/files#diff-5540e62ecf7bd041eb80e6c39296e0bf5aa925bc9da3c07b18b4afc357e88771R47